Cargo.toml

```diff
@@ -29,7 +29,7 @@ unsigned-varint = { version = "0.7.1", features = ["futures", "std"] }
 [dev-dependencies]
 async-std = { version = "1.10.0", features = ["attributes"] }
 env_logger = "0.9.0"
-libipld = { version = "0.14.0", default-features = false, features = ["dag-cbor"] }
+libipld = { version = "0.15.0", default-features = false, features = ["dag-cbor"] }
 libp2p = { version = "0.49.0", features = ["tcp", "noise", "yamux", "rsa", "async-std"] }
-multihash = { version = "0.16.1", default-features = false, features = ["blake3"] }
+multihash = { version = "0.17", default-features = false, features = ["blake3"] }
```

Comment:
I’d prefer to stay in sync with libp2p on this one, which is still on 0.16.1.

Reply:
Would have to double check, I believe this change was done for was due to the test using the reexport of `Code` from libipld while also using `MultihashDigest` from multihash itself, which caused it to failed so I bumped it up to match libipld

Reply:
just saw https://github.com/libp2p/rust-libp2p/pull/3196, so we can do this now